src/components/activities/ActivityCard.jsx

```diff
@@ -30,12 +31,17 @@ class ActivityCard extends Component {
     showUserDetails: PropType.bool,
     showLocation: PropType.bool,
     owner: PropType.string,
+    page: PropType.string,
+    handleClick: PropType.func,
```

Comment:
Considering the button is meant to be clicked, I think handleClick should be required